atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -128,7 +128,7 @@ public int execute(final AtlasDbServices services) {
             while (startRow.isPresent()) {
                 Stopwatch watch = Stopwatch.createStarted();
                 SweepResults results = sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
-                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} cells.",
+                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} stale values.",
```

Comment:
values probably isn't the best choice of word, since we are deleting more than just the value fields.

Maybe "stale versions" or "stale cells"